tests/acceptance/integrations/ContactForm7FormCest.php

```diff
@@ -122,7 +122,7 @@ private function _createContactForm7Form(AcceptanceTester $I)
 	 */
 	public function _passed(AcceptanceTester $I)
 	{
-		$I->deactivateThirdPartyPlugin($I, 'contact-form-7');
+		//$I->deactivateThirdPartyPlugin($I, 'contact-form-7');
```

Comment:
Yes; since 6.0.1, deactivating the Contact Form 7 plugin returns a 502 server error.  Happy to reinstate when they've resolved this, as it's outside of our control.